src/components/Manifest.js

```diff
@@ -20,7 +20,7 @@ const ManifestNavigation = ({ pageSizes, pageSizeLabelGenerator, statusMessageGe
   const { count } = useManifest()
   if (!count) return null
   return (
-    <Row className='align-items-center'>
+    <Row className='align-items-center mx-0'>
```

Comment:
Will this have any negative effects to the manifests that didn't have this problem?